subprojects/language-groovy/src/testFixtures/groovy/org/gradle/testing/fixture/GroovyCoverage.groovy

```diff
@@ -40,14 +43,17 @@ class GroovyCoverage {
             allVersions += GroovySystem.version
         }
 
-        ALL = allVersions
-        SUPPORTS_GROOVYDOC = allVersions.findAll {
+        // TODO: document this
+        SUPPORTS_JDK14 = allVersions - ['2.3.10', '2.5.8']
```

Comment:
Do the older versions of Groovy work with JDK14 (2.4, 2.2, etc)?

Reply:
Doh, I see now that you addressed this in the top comment.   We may want to express this as an exclusion range instead of specific version (assuming it's all 2.3.x and 2.5.x before 2.5.10).

Reply:
Handling this here https://github.com/gradle/gradle/pull/12405